src/components/site-form.tsx

```diff
@@ -290,7 +292,13 @@ export const SiteForm = ( {
 	}, [ wpVersionsStatus, dispatch ] );
 
 	const [ isAdvancedSettingsVisible, setAdvancedSettingsVisible ] = useState( false );
-
+	/*
+	useEffect( () => {
+		if ( useCustomDomain && enableHttps && ! isCertificateTrusted && ! isWindows() ) {
+			getIpcApi().openCertificate();
+		}
+	}, [ useCustomDomain, enableHttps, isCertificateTrusted ] );
+*/
```

Comment:
Should we remove this?
If not, let's add a comment to explain why it's needed.

Reply:
Yes, sorry leftover :)